### PR TITLE
feat: Org-level SRM/Jira integration TAROT-2285

### PR DESCRIPTION
**Pull request by nicklem**
Moves the Jira integration information to its own org-level integration page

### :eyes: Live preview
* https://tarot-2285-doc-document-org-level-s--docs-codacy.netlify.app/organizations/integrations/jira-integration/
* https://tarot-2285-doc-document-org-level-s--docs-codacy.netlify.app/organizations/managing-security-and-risk/

### :construction: To do
- [x] Review SRM page IA and section alignment
- [x] Concept sections
- [x] Screenshots

### CODE REVIEW

File: docs/organizations/managing-security-and-risk.md [hunk 1]
```diff
@@ -27,86 +27,89 @@ The main area of the dashboard includes five panels:
 
 Each panel shows the total count of matching items and contains a **Review** button to view a list of matching items.
 
-To limit the total counts in each panel to a specific set of severities or repositories, click the **Severity** or **Repository** drop-downs above the main area.
+When viewing the dashboard, you can:
 
-![security and risk management dashboard](images/security-risk-management-dashboard.png)
+-   Limit the total counts in each panel to a specific set of severities or repositories by clicking the **Severity** or **Repository** drop-downs above the main area.
+-   Export a list of items as a CSV file by clicking the **Export CSV** button in the top right-hand corner of the page. The exported list always includes all items, ignoring any applied filters.
```
Comment (mrfyda) [inline docs/organizations/managing-security-and-risk.md:33]: nice!

**Review by Afonsobli — COMMENTED** [review]
All good for now!

File: docs/organizations/managing-security-and-risk.md [hunk 2]
```diff
@@ -27,86 +27,89 @@ The main area of the dashboard includes five panels:
...
+To better track and address security issues, organization admins can extend access to Security and risk management by assigning the Security Manager role to organization members.
 
 The Security Manager role is independent of the Git provider role of an organization member and provides only the permissions necessary to monitor security issues, following the principle of least privilege:
 
 | Permission                                                  | Organization Admin | Security Manager           |
 |-------------------------------------------------------------|--------------------|----------------------------|
-| Access security and risk management                         | Yes                | Yes                        |
+| Access Security and risk management                         | Yes                | Yes                        |
 | Access associated issues (Codacy repositories, Jira issues) | Platform-dependent | Keeps original permissions |
 | Manage integrations                                         | Yes                | Yes                        |
```
[the remark this answers is not in the document]
Comment (nicklem) [reply docs/organizations/managing-security-and-risk.md:114]: Although we never "officially" refer to Slack as a SRM integration in the docs 🤔 

Should we start thinking about creating a new doc page only for these dynamic permissions? IIRC these will eventually outgrow SRM

[the remark this answers is not in the document]
Comment (mrfyda) [reply docs/organizations/managing-security-and-risk.md:114]: I'm good if you think it's clear in this context that "Manage integrations" only refers to the Jira integration (i.e. visible in the SRM configuration page) 👍

[the remark this answers is not in the document]
Comment (nicklem) [reply docs/organizations/managing-security-and-risk.md:114]: We may still show a link to the Slack integration on the SRM config page (see [this thread](https://www.figma.com/file/Zw8KLyYC3Erg3ccYyb9lsz?node-id=2550:22109&mode=design#552271082)). That may confuse users, since it associates Slack with SRM...

I'll see how it looks once implemented and clarify if needed.

File: docs/organizations/integrations/jira-integration.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Organization Jira integration for Security and risk management
```
Comment (mrfyda) [inline docs/organizations/integrations/jira-integration.md:1]: opened codacy-spa#1410 which links to this page instead

File: docs/organizations/managing-security-and-risk.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nicklem) [reply docs/organizations/managing-security-and-risk.md]: I suggest going with "Manage the Jira integration".

It will still be unclear what Slack is doing there. I don't want to block this though and will review the entirety of the page in a follow-up task ([DOCS-584](https://codacy.atlassian.net/browse/DOCS-584)).

**Review by joanasteodoro — APPROVED** [review]
LGTM!

**Review by mrfyda — APPROVED** [review]

File: docs/organizations/integrations/jira-integration.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+## Installing the Jira integration
+
+!!! note
+    -   Before installing the Jira integration, make sure to tag the Jira issues you wish to import with the label **security** (case-insensitive).
+
+    -   Use a Jira account with admin permissions when installing this integration. This lets Codacy access all issues, since the integration inherits the permissions of the account that installs it.
+
+To install the Jira integration:
+
+1.  Open your organization **Integrations**, page **Jira**, and click **Install Jira** to proceed to Atlassian's website.
```
Comment (claudiacodacy) [inline docs/organizations/integrations/jira-integration.md:16]: @nicklem here, instead of telling users to install right away, I would first talk about the things users must do for Jira to work properly. so before installing, users should review the tasks they would like to monitor to add the security label, a pro tip would be to add that label as a default to all security issues, and make sure the priorities used on those issues are similar to the ones we use to track severity.

to summarize, I would put a bullet before the instal that you have here, and mention these topics before users click Install

File: docs/organizations/integrations/jira-integration.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+    ![Security and risk management Jira integration installation](images/jira-integration-srm-install.png)
+
+1.  On Atlassian's website, authorize Codacy. Once successful, you're redirected back to Codacy.
+
+After installing, Codacy imports all open Jira issues created up to 90 days before the integration.
```
Comment (claudiacodacy) [inline docs/organizations/integrations/jira-integration.md:22]: all open Jira issues that fit the criteria described above

**Review by Afonsobli — APPROVED** [review]

File: docs/organizations/integrations/jira-integration.md (hunk 4 above)
Comment (nicklem) [reply docs/organizations/integrations/jira-integration.md:16]: OK, I'm moving the prerequisites lower down added visibility ([commit](https://github.com/codacy/docs/pull/1843/commits/a3fc093a48579ed475e82d1e4024af58705e953e)).

On a related note, I'm noticing a difference in how we label the Jira priorities. Are they equivalent or is one of these wrong? Thanks 🙏 

* On the [Integrations](https://app.codacy.com/organizations/gh/troubleshoot-codacy/integrations/jira) page: **Highest**, High, Medium and Low
* On the [SRM Configuration](https://app.codacy.com/organizations/gh/troubleshoot-codacy/security/configuration) page and docs: **Blocker**, High, Medium and Low

Comment (nicklem) [reply docs/organizations/integrations/jira-integration.md:16]: cc @claudiacodacy @mrfyda

Comment (claudiacodacy) [reply docs/organizations/integrations/jira-integration.md:16]: ufff!! great catch Nico! It's my fault, i think i used an older version of the mockups...

@joanasteodoro could you help me here by replacing **Highest** -> **Blocker**? The SRM is the most accurate source of truth, I'll update the mockups right away

File: docs/organizations/integrations/jira-integration.md (hunk 5 above)
Comment (nicklem) [reply docs/organizations/integrations/jira-integration.md:25]:
```suggestion
After installing, Codacy imports all open Jira issues labeled **security** and created up to 90 days before the integration.
```

File: docs/organizations/managing-security-and-risk.md [hunk 6]
```diff
@@ -3,9 +3,9 @@
 !!! info "This is a preview feature"
     This is a new Codacy feature and <span class="skip-vale">we're</span> continuing to improve it.
 
-The security and risk management feature helps you <span class="skip-vale">quickly</span> identify, track, and address security issues by automatically opening time-bound, prioritized action items whenever Codacy detects security issues in your organization repositories or in your connected Jira Cloud instance.
+The Security and risk management feature helps you <span class="skip-vale">quickly</span> identify, track, and address security issues by automatically opening time-bound, prioritized action items whenever Codacy detects security issues in your organization repositories or in your [connected Jira instance](./integrations/jira-integration.md).
```
[the remark this answers is not in the document]
Comment (nicklem) [reply docs/organizations/managing-security-and-risk.md:6]: It's capitalized "Security and risk" in the UI and we generally use sentence case (capitalize only the first word).

I personally prefer title case as you suggest, since it clarifies all the words that make up the feature, but we'll have to go with sentence case here for consistency.

```suggestion
The Security and risk management (SRM) feature helps you <span class="skip-vale">quickly</span> identify, track, and address security issues by automatically opening time-bound, prioritized action items whenever Codacy detects security issues in your organization repositories or in your [connected Jira instance](./integrations/jira-integration.md).
```

File: docs/organizations/managing-security-and-risk.md [hunk 7]
```diff
@@ -3,9 +3,9 @@
...
 
-Under security and risk management, you can find the following pages to help you monitor your security issues:
+Under Security and risk management, you can find the following pages to help you monitor your security issues:
```
[the remark this answers is not in the document]
Comment (nicklem) [reply docs/organizations/managing-security-and-risk.md:8]: Again, here unfortunately consistency trumps usability. We can review and start capitalizing all words, but has to be done consistently at the product level.

File: docs/organizations/integrations/jira-integration.md (hunk 4 above)
Comment (mrfyda) [reply docs/organizations/integrations/jira-integration.md:16]: I think it should show Jira's default priorities: https://confluence.atlassian.com/jirakb/translating-the-priority-field-1189801690.html

Comment (joanasteodoro) [reply docs/organizations/integrations/jira-integration.md:16]: I agree with Rafael, if we use the same language it would be less confusing to do the translation @claudiacodacy

Comment (nicklem) [reply docs/organizations/integrations/jira-integration.md:16]: 👍 [updated](https://github.com/codacy/docs/pull/1843/files#diff-30a735ab496ba1650575435a2195b32444487dbb269bb4b685b2e1ad8c7ebe53R182)

**Comment by joanasteodoro** [comment]
The UI was updated (again 🙈) regarding the Jira's priority issues. Other than that, we can merge it, wdyt?

File: docs/organizations/managing-security-and-risk.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Afonsobli) [reply docs/organizations/managing-security-and-risk.md:6]: Makes sense to me!
